algebraic_connectivity_notebook.ipynb

```diff
@@ -112,7 +111,7 @@
     "    \"name\": \"MIDSdataset\",\n",
     "    \"selected_graphs\": str(selected_graph_sizes),\n",
     "    \"split\": 0.8,\n",
-    "    \"batch_size\": 6000,\n",
+    "    \"batch_size\": 1,\n",
```

Comment:
Batch size is now defined as a ratio (1 - fit the whole training dataset in one batch, 0.5 - put half of the data in one batch, etc.). The number of batches is ~ 1 / `batch_size`.

This should be easier than thinking about how many graphs are in the dataset and how many of them will fit in one batch.

We can revert this change if it's not desirable for some reason.